platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateSigningRequest.java

```diff
@@ -62,6 +68,8 @@
 
     private X500Name subjectName;
 
+    private Set<GeneralName> subjectAltNames = new LinkedHashSet<>(8); // to preserve order
```

Comment:
I don't expect we will ever have a lot of SANs. 8 makes it more optimal then the default size used by the default ctors which tends to allocate too much memory for nothing.